plan.sh

```diff
@@ -1,16 +1,24 @@
 pkg_name=erlang18
 pkg_origin=core
-pkg_version=18.3
+pkg_version=18.3.4.11
 pkg_description="A programming language for massively scalable soft real-time systems."
 pkg_upstream_url="http://www.erlang.org/"
 pkg_dirname=otp_src_${pkg_version}
 pkg_license=('Apache-2.0')
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
-pkg_source=http://www.erlang.org/download/otp_src_${pkg_version}.tar.gz
+pkg_source=https://api.github.com/repos/erlang/otp/tarball/refs/tags/OTP-${pkg_version}
 pkg_filename=otp_src_${pkg_version}.tar.gz
-pkg_shasum=fdab8129a1cb935db09f1832e3a7d511a4aeb2b9bb3602ca6a7ccb9730d5c9c3
+pkg_shasum=6f710780ff83e672aee9e6dc5739853897dc0b36a52660d67a6997fe5a849c80
 pkg_deps=(core/glibc core/zlib core/ncurses core/openssl core/sed)
-pkg_build_deps=(core/coreutils core/gcc core/make core/openssl core/perl core/m4)
+pkg_build_deps=(
+	core/coreutils
+       	core/gcc
+       	core/make
+       	core/openssl
+       	core/perl
+       	core/m4
+	core/autoconf/2.69
```

Comment:
Need to discuss deps change and pinned version .

Reply:
I think we will need to pin to 2.69. All solutions I found on google were exactly that: use 2.69. Because this is a build dep of erlang and because autoconf is in almost all cases only going to be depended on as a build dep, this should not cause version conflicts. It would only be an issue when building erlang AND if one of erlangs other build/runtime deps had a runtime dep on autoconf (very unlikely). So if this did blow up, it would only do so while building erlang.